code/game/machinery/_machinery.dm

```diff
@@ -320,6 +324,8 @@ Class Procs:
 
 /obj/machinery/ui_act(action, params)
 	add_fingerprint(usr)
+	if(isliving(usr) && in_range(src, usr))
```

Comment:
Is there not a range check anywhere before this?

Reply:
As far as I know, nope, the range check is there so it doesn't do clicking sounds while staying out of the range with the machine/computer that is being interacted with.